src/main/webapp/jsp/instructorFeedbackResultsByQuestion.jsp

```diff
@@ -215,9 +252,27 @@
                                           		    if (questionDetails.shouldShowNoResponseText(prevGiver, possibleReceiver, question)) {
                                 %>
                                                         <tr class="pending_response_row">
-                                                            <td class="middlealign color_neutral"><%=data.bundle.getFullNameFromRoster(prevGiver)%></td>
+                                                            <td class="middlealign color_neutral">
+                                                                <%if (question.giverType == FeedbackParticipantType.SELF || question.giverType == FeedbackParticipantType.STUDENTS) {%>
+                                                                <div class="profile-pic-icon-hover" data-link="<%=data.getProfilePictureLink(prevGiver)%>">
+                                                                    <%=data.bundle.getFullNameFromRoster(prevGiver)%>
+                                                                    <img src="" alt="No Image Given" class="hidden profile-pic-icon-hidden">
+                                                                </div>       
+                                                                <%} else {%>
+                                                                <%=data.bundle.getFullNameFromRoster(prevGiver)%>
+                                                                <%}%>                                                            
+                                                            </td>
                                                             <td class="middlealign color_neutral"><%=data.bundle.getTeamNameFromRoster(prevGiver)%></td>
-                                                            <td class="middlealign color_neutral"><%=data.bundle.getFullNameFromRoster(possibleReceiver)%></td>
+                                                            <td class="middlealign color_neutral">
+                                                                <%if (question.recipientType == FeedbackParticipantType.SELF || question.recipientType == FeedbackParticipantType.STUDENTS || question.recipientType == FeedbackParticipantType.OWN_TEAM_MEMBERS || question.recipientType == FeedbackParticipantType.OWN_TEAM_MEMBERS_INCLUDING_SELF) {%>
```

Comment:
hey! you could consider extracting this check into a boolean function in the question object. smth like: isRecipientAStudent()